src/core/issues-notes.js

```diff
@@ -118,7 +119,20 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          const labelsGroup = Array.from(ghIssue.labels)
+            .map(createLabels)
```

Comment:
nit: rename "createLabels" to "createLabel"

Reply:
yep